mysql/cloudmysql/cloudmysql.go

```diff
@@ -56,7 +58,7 @@ type Params struct {
 }
 
 // Open opens a Cloud SQL database.
-func Open(ctx context.Context, certSource proxy.CertSource, params *Params) (*sql.DB, error) {
+func Open(certSource proxy.CertSource, params *Params) (*sql.DB, error) {
```

Comment:
Hmm, it's probably worth checking with whoever added `Open` (@zombiezen or @shantuo probably) about why it took a content, before removing it. Maybe there's a good reason.

Reply:
When in doubt, it's usually better to plumb a `Context`. Especially if you're doing something that will probably do networky things, it's usually better to include it.

Reply:
In this case, is that because we might need it later and we don't want to break existing calls to cloudmysql.Open?

Reply:
Also, done.

Reply:
Precisely.